packages/sources/tradingeconomics-test/src/index.ts

```diff
@@ -0,0 +1,32 @@
+import { expose, ServerInstance } from '@chainlink/external-adapter-framework'
+import { price } from './endpoint'
+import { customSettings } from './config'
+import { PriceAdapter } from '@chainlink/external-adapter-framework/adapter'
+import symbols from './config/symbols.json'
+
+export const adapter = new PriceAdapter({
+  name: 'TRADINGECONOMICS',
+  defaultEndpoint: price.name,
+  customSettings,
+  overrides: symbols['tradingeconomics'],
```

Comment:
I noticed that `includes` isn't included as a parameter here. I think the one from v2 may not be compatible with v3 so we'll need to see if we can rework it